src/core/style/layout.scss

```diff
@@ -153,37 +150,43 @@
   // (0-20) * 5 = 0-100%
   @for $i from 0 through 20 {
     [#{$flexName}="#{$i * 5}"] {
-      flex: 0 0 #{$i * 5 + '%'};
+      flex: #{($i * 5)/100};
     }
     [layout="row"] > [#{$flexName}="#{$i * 5}"] {
       max-width: #{$i * 5 + '%'};
+      min-width: #{$i * 5 + '%'};
     }
     [layout="column"] > [#{$flexName}="#{$i * 5}"] {
       max-height: #{$i * 5 + '%'};
+      min-height: #{$i * 5 + '%'};
     }
   }
 
   [#{$flexName}="33"], [#{$flexName}="34"] {
-    flex: 0 0 33.33%;
+    flex: 0.33;
```

Comment:
It might work in this particular case, but flex 0.33 is quite arbitrary and won't work well with other values.
E.g. imagine a `[flex="33"]` next to a `[flex]`. One would expect that the former takes up 33% of the container and the latter takes up the rest. In practice, though, the former would take up 33/133 (~25%) of the container and the latter would take up 100/133 (~75%).